nomt/src/sys/unix.rs

```diff
@@ -3,21 +3,29 @@
 use std::{fs::File, os::fd::AsRawFd as _};
 
 pub fn try_lock_exclusive(file: &File) -> std::io::Result<()> {
-    unsafe {
-        if libc::flock(file.as_raw_fd(), libc::LOCK_EX | libc::LOCK_NB) == -1 {
-            Err(std::io::Error::last_os_error())
-        } else {
-            Ok(())
-        }
-    }
+    cvt_r(|| unsafe { libc::flock(file.as_raw_fd(), libc::LOCK_EX | libc::LOCK_NB) }).map(drop)
 }
 
 pub fn unlock(file: &File) -> std::io::Result<()> {
-    unsafe {
-        if libc::flock(file.as_raw_fd(), libc::LOCK_UN) == -1 {
+    unsafe { cvt_r(|| libc::flock(file.as_raw_fd(), libc::LOCK_UN)).map(drop) }
+}
+
+pub(super) fn cvt_r<F>(mut f: F) -> std::io::Result<i32>
```

Comment:
what does CVT mean?

Reply:
stands for [convert](https://github.com/badboy/cvt/blob/45d62f6f298a69bab6cd573c73eee2d13b7a37af/src/lib.rs#L2C5-L2C12).